CMakeLists.txt

```diff
@@ -67,86 +48,101 @@ set( WIN32_BOOTSTRAP_SOURCES
 
 # ==============================================================================
 
-function(_COMMON_PROJECT_OPTIONS)
-	include_directories(${INCLUDE_DIR} ${LIB_INCLUDE_DIR})
-	link_directories(${LIB_DIR}/boost/lib/)
-
-	add_compile_options(-std:c++latest)
+function(_COMMON_PROJECT_OPTIONS TARGET_NAME)
```

Comment:
I added a target as argument to avoid to globally declared include directories or linked libraries